src/fee_estimator.rs

```diff
@@ -61,23 +81,30 @@ where
 		}
 
 		let confirmation_targets = vec![
-			ConfirmationTarget::OnChainSweep,
-			ConfirmationTarget::MinAllowedAnchorChannelRemoteFee,
-			ConfirmationTarget::MinAllowedNonAnchorChannelRemoteFee,
-			ConfirmationTarget::AnchorChannelFee,
-			ConfirmationTarget::NonAnchorChannelFee,
-			ConfirmationTarget::ChannelCloseMinimum,
-			ConfirmationTarget::OutputSpendingFee,
+			ConfirmationTarget::OnchainPayment,
+			ConfirmationTarget::ChannelFunding,
+			LdkConfirmationTarget::OnChainSweep.into(),
+			LdkConfirmationTarget::MinAllowedAnchorChannelRemoteFee.into(),
+			LdkConfirmationTarget::MinAllowedNonAnchorChannelRemoteFee.into(),
+			LdkConfirmationTarget::AnchorChannelFee.into(),
+			LdkConfirmationTarget::NonAnchorChannelFee.into(),
+			LdkConfirmationTarget::ChannelCloseMinimum.into(),
+			LdkConfirmationTarget::OutputSpendingFee.into(),
 		];
+
 		for target in confirmation_targets {
 			let num_blocks = match target {
-				ConfirmationTarget::OnChainSweep => 6,
-				ConfirmationTarget::MinAllowedAnchorChannelRemoteFee => 1008,
-				ConfirmationTarget::MinAllowedNonAnchorChannelRemoteFee => 144,
-				ConfirmationTarget::AnchorChannelFee => 1008,
-				ConfirmationTarget::NonAnchorChannelFee => 12,
-				ConfirmationTarget::ChannelCloseMinimum => 144,
-				ConfirmationTarget::OutputSpendingFee => 12,
+				ConfirmationTarget::OnchainPayment => 6,
+				ConfirmationTarget::ChannelFunding => 12,
```

Comment:
Seems like in practice the values are the same as before. Is this change to allow us to vary them in the future?